include/functions/math.hpp

```diff
@@ -33,4 +30,42 @@ BigInt big_pow10(size_t exponent) {
     return BigInt("1" + std::string(exponent, '0'));
 }
 
+/*
+    pow
+    ---
+    Return a BigInt equal to my_big_int^x
+    NOTE: x could be BigInt, long long, or string
+*/
+BigInt pow(BigInt my_big_int, long long x) {
```

Comment:
The parameters should be passed as constant references (`const Type&`).

Reply:
Done